s4/s4.py

```diff
@@ -619,18 +620,18 @@ def SSMInit(N):
 # Here we add a block that pairs a call to an SSM with
 # dropout and a linear projection.
 
-
 class SequenceBlock(nn.Module):
     layer: nn.Module
     l_max: int
     dropout: float
     d_model: int
     training: bool = True
     decode: bool = False
+    pre_norm: bool = True
 
     def setup(self):
+        self.norm = nn.BatchNorm(use_running_average=not self.training)
```

Comment:
can you have norm type be an option? Be good not to lose layernorm.

Reply:
could be hard -- different shapes compared to LN, but can check and try.

Reply:
I don't think that's true.